exo-go/src/docker_compose/index.go

```diff
@@ -18,3 +18,19 @@ func KillAllContainers(cwd string, log func(string)) (*processHelpers.Process, e
 	process.SetStdoutLog(log)
 	return process, process.Start()
 }
+
+// PullAllImages pulls all the docker images defined in docker-compose.yml
+func PullAllImages(dockerComposeLocation string, log func(string)) (*processHelpers.Process, error) {
+	process := processHelpers.NewProcess("docker-compose", "pull")
+	process.SetDir(dockerComposeLocation)
+	process.SetStdoutLog(log)
+	return process, process.Start()
```

Comment:
The use of each of these calls Wait right after. Thoughts on updating these functions to wait for the process to finish and thus just return an error. Same for `BuildAllImages`